addon/styles/_frost-textarea.scss

```diff
@@ -6,8 +6,8 @@ $disabled-border: 1px solid $frost-color-input-disabled-border;
 $disabled-bg: $frost-color-input-disabled-bg;
 
 .frost-textarea {
-  display: flex;
-  align-items: flex-start;
+  display: inline-block;
```

Comment:
We need to be careful with this, flex is a block display (e.g. the next peer element will be on the next line) so if we're moving from flex we should move to `block`, any reason why `inline-block` was chosen?